### PR TITLE
Add a bio for @mollings

### PR DESCRIPTION
**Pull request by mollings**
This Pull Request adds a bio for @mollings as requested in #18.

@ryanjames28 was supposed to do this as assigned, but couldn't be trusted to complete it on time.

### CODE REVIEW

File: docs/mollings.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# mollings
+
+**Name:** Scott Molling
```
Comment (ghost) [inline docs/mollings.md:3]: You're the greatest Scott

**Comment by ryanjames28** [comment]
i think these look good :smirk_cat:

File: docs/mollings.md (hunk 1 above)
Comment (ghost) [inline docs/mollings.md:3]: thats garbage you're putting words in my mouth

**Comment by alcala21** [comment]
Good job, @mollings.

**Comment by mollings** [comment]
:+1: 

:shipit: :smile_cat:

**Pull request merged**